src/test/resources/org/openmrs/uitestframework/test.properties

```diff
@@ -4,5 +4,5 @@ login.password=Admin123
 login.location=Pharmacy
 webdriver=firefox
 login.auto=false
-headless=true
+headless=false
```

Comment:
Hi @Ndacyayisenga-droid when you edit the test.properties file for your test simulation, you don't have to push these changes, actually you should always be reverting to the original files content before adding your change to git.